src/components/TriggerEditor/index.tsx

```diff
@@ -360,6 +360,7 @@ export default ({
     }
     let trigger: AstarteTrigger | null = null;
     try {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
i'd rather keep this as a warning for now, reminding us we're not finished yet.
We can model/fix this on a future PR, but we might forget about this is we just disable the warning

Reply:
I can agree, I left the warning active